core/kits/manager.php

```diff
@@ -179,6 +185,18 @@ public static function create_default_kit() {
 		return $id;
 	}
 
+	public function revert( $new_kit_id, $active_kit_id, $previous_kit_id ) {
```

Comment:
Try and find a better name for new_kit_id

Reply:
Done